PF/Assignment1/anshitaarya/prg2.java

```diff
@@ -0,0 +1,41 @@
+package anshitaarya;
+
+public class prg2 {
+	 public static int[] removeDuplicates(int[] duplicate){
+         int l= duplicate.length;
+         int i,j,m=1,count;
+        
+         int[] dup= new int[l];
+         dup[0]=duplicate[0];
+         for(i=1;i<l;i++)
+         {
+        	 count=0;
+        	for(j=0;j<i;j++)
+        	{
+        		if(duplicate[i]==dup[j])
```

Comment:
ACTION: Logic is not correct, If there are duplicate elements then size of dup array will not increase in that case size of dup array will be less than the current position(i) of duplicate array and in if condition dup[j] will give you arayindexoutofbound exception.
Test this method for array 2,2,2,4,5,6 . It will fail